### PR TITLE
Add new category "standard volume per standard volume"

### PR DESCRIPTION
**Pull request by oliveira-mauricio**
Move "sm3/sm3" unit from "volume per volume" to the new category

### CODE REVIEW

**Comment by oliveira-mauricio** [comment]
> * Please add some tests doing a few known conversions; this is mostly a smoke tests to ensure the conversion coefficients are correct
> * Update `CHANGELOG.rst`

Added the changelog and the test. I'm counting on @arthursoprana to review the conversions (and test), I simply copied the conversion formulas from "standard volume per volume" as he suggested, but since that category is not tested, I'm not sure the conversions are ok too.

File: src/barril/units/posc.py [hunk 1]
```diff
@@ -204,6 +204,11 @@ def FillUnitDatabaseWithPosc(db=None, fill_categories=True, override_categories=
     db.AddUnitBase(
         "standard volume per volume", "std cubic metres, 15 deg C/cubic metre", "scm(15C)/m3"
     )
+    db.AddUnitBase(
+        "standard volume per standard volume",
+        "std cubic metres, 15 deg C/std cubic metre",
```
Comment (arthursoprana) [inline src/barril/units/posc.py]: I think here it should be

```suggestion
        "std cubic metres, 15 deg C/std cubic metre, 15 deg C",
```

File: src/barril/units/posc.py [hunk 2]
```diff
@@ -204,6 +204,11 @@ def FillUnitDatabaseWithPosc(db=None, fill_categories=True, override_categories=
...
+        "scm(15C)/sm3",
```
Comment (arthursoprana) [inline src/barril/units/posc.py]:
```suggestion
        "scm(15C)/scm(15C)",
```

File: src/barril/units/posc.py [hunk 3]
```diff
@@ -12094,6 +12089,86 @@ def FillUnitDatabaseWithPosc(db=None, fill_categories=True, override_categories=
...
+    f_base_to_unit = MakeBaseToCustomary(0.0, 1, 1.0, 0.0)
+    db.AddUnit(
+        "standard volume per standard volume",
+        "std cubic centimetres/std cubic centimetres",
+        "scm3/scm3",
+        f_base_to_unit,
+        f_unit_to_base,
+        default_category=None,
+    )
+    f_unit_to_base = MakeCustomaryToBase(0.0, 1, 1.0, 0.0)
```
Comment (arthursoprana) [inline src/barril/units/posc.py]:
```suggestion
    f_unit_to_base = MakeCustomaryToBase(0.0, 0.1589873, 1.0, 0.0)
    f_base_to_unit = MakeBaseToCustomary(0.0, 0.1589873, 1.0, 0.0)
```

File: src/barril/units/posc.py [hunk 4]
```diff
@@ -12094,6 +12089,86 @@ def FillUnitDatabaseWithPosc(db=None, fill_categories=True, override_categories=
...
+    f_base_to_unit = MakeBaseToCustomary(0.0, 1, 1.0, 0.0)
```
Comment (arthursoprana) [inline src/barril/units/posc.py]:
```suggestion
	f_base_to_unit = MakeBaseToCustomary(0.0, 0.1589873, 1.0, 0.0)
```

File: CHANGELOG.rst [hunk 5]
```diff
@@ -1,3 +1,9 @@
+1.7.2 (2020-01-10)
```
Comment (nicoddemus) [inline CHANGELOG.rst]:
```suggestion
1.8.0 (2020-01-10)
```

File: src/barril/units/_tests/test_posc.py [hunk 6]
```diff
@@ -410,3 +410,16 @@ def testDensityDerivativePerTemperatureUnitConversion():
         "density derivative in respect to temperature", 1, "kg/m3.K"
     )
     assert density_derivative_per_temperature.GetValue("kg/m3.degC") == 1.0
+
+
+def testStandardVolumePerStandardVolumeSmoke():
+    standard_per_standard = units.Scalar("standard volume per standard volume", 1, "scm(15C)/sm3")
```
Comment (oliveira-mauricio) [inline src/barril/units/_tests/test_posc.py]:
```suggestion
    standard_per_standard = units.Scalar("standard volume per standard volume", 1, "scm(15C)/scm(15C)")
```

File: src/barril/units/posc.py [hunk 7]
```diff
@@ -14791,6 +14865,22 @@ def FillUnitDatabaseWithPosc(db=None, fill_categories=True, override_categories=
                 "scf/bbl",
             ],
         )
+        db.AddCategory(
+            "standard volume per standard volume",
+            "standard volume per standard volume",
+            override=override_categories,
+            valid_units=[
+                "scm(15C)/sm3",
```
Comment (oliveira-mauricio) [inline src/barril/units/posc.py]:
```suggestion
                "scm(15C)/scm(15C)",
```

File: CHANGELOG.rst [hunk 8]
```diff
@@ -1,3 +1,9 @@
+1.8.0 (2020-01-10)
+------------------
+
+* Add new category: "standard volume per standard volume"
+* Move unit ``sm3/sm3`` from "volume per volume" to "standard volume per standard volume"
```
Comment (prusse-martin) [inline CHANGELOG.rst]: Let's hope no one was using this unit with the former quantity =)

Comment (oliveira-mauricio) [reply CHANGELOG.rst]: Yep, but we believe only ALFAsim was using.

**Pull request merged**